publish/adapter.debug.js

```diff
@@ -1063,3 +1063,22 @@ if (navigator.mozGetUserMedia) {
     AdapterJS.WebRTCPlugin.defineWebRTCInterface,
     AdapterJS.WebRTCPlugin.pluginNeededButNotInstalledCb);
 }
+
+// It's just for webRTCReady function
+AdapterJS.webRTCReady = function (callback) {
+  if (typeof callback !== 'function') {
+    throw new Error('Callback provided is not a function');
+  }
+
+  if (window.webrtcDetectedBrowser !== 'safari' && window.webrtcDetectedBrowser !== 'IE') {
+    setTimeout(function () {
```

Comment:
@serrynaimo correct implementation should be waiting for document.readyState to be completed. To be amended.